distribution/changelog.txt

```diff
@@ -187,7 +187,7 @@
 - Feature: Finance window accessible from toolbar (enabled in settings).
 - Feature: Research and development / research funding now accessible as a stand alone window without the requirement of the finances window.
 - Feature: Extra viewport windows.
-- Feature: Park window viewport is resizable.
+- Feature: Park window viewport is resizeable.
```

Comment:
Yea, I read some quick references on it. Seems like in computing resizable is fine, however I was going off of personal habit and a spellchecker. Even typing resizable in this window now it comes up red. I know it's not wrong per say, but it does bug me. Do you think it should be reverted back or is it fine with my change?